README.md

```diff
@@ -132,4 +133,13 @@ Planned.
 ## Manually
 
 It's still possible to compile the project by referencing every source files.
-The `-betterC` switch is optional, but recommended.
\ No newline at end of file
+The `-betterC` switch is optional, but recommended.
+
+## Notes on fuzzing
+In order to preform a fuzz, ldc version 1.0.0 or newer and AFL 2.50 or newer are required, additionally the LLVM version that ldc and the library `afl-llvm-pass.so` have been built with must be the same.
+
+To fuzz, export the environment variable `AFL_ROOT` to the location where `afl-llvm-pass.so` is located, then build with `dub -d afl`.
+
+Then create two directories, `findings` and `testcases`, after that populate `testcases` with files you wish to test with.
```

Comment:
I think it'd be better to explain that AFL needs the files `testcases/*` to mutate them, if they're unfamiliar with AFL (i.e. me!)

Also, would it be better to add `findings` and `testcases` into `.gitignore`?